src/main/java/mate/academy/lib/Injector.java

```diff
@@ -33,7 +33,7 @@ public static Injector getInstance(String mainPackageName) {
         return injector;
     }
 
-    public Object getInstance(Class<?> certainInterface) {
+    public static Object getInstance(Class<?> certainInterface) {
```

Comment:
This method is public and static, but it's not used outside of this class. Consider changing its access modifier to private.